narwhal/primary/src/grpc_server/configuration.rs

```diff
@@ -84,43 +83,8 @@ impl Configuration for NarwhalConfiguration {
     #[allow(clippy::mutable_key_type)]
     async fn new_network_info(
         &self,
-        request: Request<NewNetworkInfoRequest>,
```

Comment:
As far as I remember this was a request from Celo/Sommelier. The idea is that networking information might not be available from the beginning and change down the line. It seems that with this PR we are removing completely this ability. Apart from that, I am not convinced that this endpoint was working correctly even before either. It seems that we do swap the committee , but never update the actual networking info via anemo. That being said I would at least add here a TODO to indicate a breaking behaviour